draco/core/containers.py

```diff
@@ -603,3 +603,46 @@ def freq(self):
     @property
```

Comment:
where is this property 'map' coming from? In master there seems to be a function before class 'BeamPertubation'. 
I know Richard suggested to do some 'git magic'(which I am not 100% sure how to do) but you could simply try to resolve the conflict and delete the stuff that does not belong!